crafters/image/TorchObjectDetectionSegmenter/requirements.txt

```diff
@@ -1,2 +1,2 @@
 torchvision>=0.3.0
```

Comment:
Shouldn't `torchvision` also be a fixed requirement?

Reply:
oh yes, missed that